remotecommanddispatcher/src/test/java/com/tealium/remotecommanddispatcher/RemoteCommandParserTests.kt

```diff
@@ -129,7 +244,7 @@ class RemoteCommandParserTest {
         val eventKey = "event"
         val lookup = mapOf("level" to "$eventKey.fb_level")
         val result = RemoteCommandParser.mapDispatch(dispatch, lookup)
-
+        println("result: $result")
         assertTrue(result.containsKey(eventKey))
         assertEquals(1, result.keys.count())
         (result[eventKey] as? Map<*, *>)?.let {
```

Comment:
suggest we rework these at some point - they do function as expected, but with the null checking and safe casts, it's possible in future that these assertions never get run if something in the test code above it was updated.